podvm/files/etc/systemd/system/multi-user.target.wants/kata-opa.service

```diff
@@ -0,0 +1 @@
+../kata-opa.service
```

Comment:
this will always start OPA, even if no support is built into kata. Can we `systemctl enable` this unit depending on the `AGENT_POLICY` setting in the packer build script?